pkg/blob/nodeserver.go

```diff
@@ -213,15 +224,28 @@ func (d *Driver) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRe
 
 	klog.V(2).Infof("target %v\nprotocol %v\n\nvolumeId %v\ncontext %v\nmountflags %v\nmountOptions %v\nargs %v\nserverAddress %v",
 		targetPath, protocol, volumeID, attrib, mountFlags, mountOptions, args, serverAddress)
-	cmd := exec.Command("blobfuse", strings.Split(args, " ")...)
 
-	cmd.Env = append(os.Environ(), "AZURE_STORAGE_ACCOUNT="+accountName)
-	cmd.Env = append(cmd.Env, "AZURE_STORAGE_BLOB_ENDPOINT="+serverAddress)
-	cmd.Env = append(cmd.Env, authEnv...)
+	blobfuseProxyEndpoint := "unix:///var/lib/kubelet/plugins/blobfuseproxy.sock"
+	transportOption := grpc.WithInsecure()
 
-	output, err := cmd.CombinedOutput()
+	klog.V(2).Infof("NodeStageVolume: sending GRPC call to blobfuseproxy")
+	cc1, err := grpc.Dial(blobfuseProxyEndpoint, transportOption)
+	if err != nil {
+		klog.V(2).Info("cannot dial server: ", err)
```

Comment:
is it possible to fall back to use original way is blobfuseproxy is not installed on the node

Reply:
sure.